src/tests/path.test.ts

```diff
@@ -64,6 +65,14 @@ describe("Cloning", () => {
           encoding: "utf8",
         });
 
+        const mythOutput = execSync(
+          `myth analyze --execution-timeout 10 -c ${rootContractPath}`,
+          {
+            cwd: projectPath,
+            encoding: "utf8",
+          }
+        );
```

Comment:
Why do you execute mytril here?

Reply:
Oh, sorry. That's some leftover code from writing tests 😁